sections/user.zsh

```diff
@@ -16,7 +16,14 @@ SPACESHIP_USER_COLOR_ROOT="${SPACESHIP_USER_COLOR_ROOT:="red"}"
 # Section
 # ------------------------------------------------------------------------------
 
-# If user is root, then paint it in red. Otherwise, just print in yellow.
+# --------------------------------------------------------------------------
```

Comment:
LGTM, but I'd rather moved this table under _Configuration_ section since it's related to options.

Reply:
Done